custom-connectors/HR Cloud/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
```

Comment:
@HRCIntegrationsTeam Please let us know the reason for `no_auth`

Reply:
Our API has an authentification process with two values in the headers: customer_key and customer_secret. 
We tried all the authentication options you offered, but none of them fit this specific flow we have.
Our process is the closes to the API key auth, but has two values instead of the one. 

Customers will still have to authenticate themselves before making the request. We ask them to input auth values as headers, instead of using built in auth method.

If you have a better solution how to solve our authentication through your process, we are willing to make the changes.

Reply:
@diegobernardes You can use `Basic` authentication method where you can pass Key and Secret like username and password.

Reply:
Hi,
we tried that and it does not fit our needs. When we use basic auth it creates an authorization header with a Base64 encoded combination of username and password (or in our case, combines customer key and secret).

Our API expects two separate headers named "Customer_Key" and "Customer_Seceret". As mentioned above this is similar to API keys, but there are two instead of the one.

Reply:
@mamurshe  Could you please suggest partner with suitable authentication type for their connector

Reply:
@sriyen-msft @mamurshe 
Any info on this? We resolved all other comments and were wondering when could we deal with this.
Our system still authenticates users, we are just doing it through headers instead of going through built-in auth methods. 
Our users are familiar with this authentication method and are using it in other systems.

Reply:
@diegobernardes providing auth as part of the connector is a security risk, since if one of you customer shares the connector, the key is also shared with it. By design the connection and connectors are separated to avoid such leaks.
Since you own the api can your change it to accept say a api key that constitutes of you key and secret? Or accept basic auth with the key as username and secret as password?

Reply:
@diegobernardes providing auth as part of the connector is a security risk, since if one of you customer shares the connector, the key is also shared with it. By design the connection and connectors are separated to avoid such leaks.
Since you own the api can your change it to accept say a api key that constitutes of you key and secret? Or accept basic auth with the key as username and secret as password?

Reply:
@mamurshe @sriyen-msft 
we changed an authentification to use username and password.
This is not the best option for us because our users commonly prefer to use customer keys and customer secrets, but if there is no other option we can support this way too.

Reply:
Thank you for fixing the issue. I have few more comments.